ui/app/AppLayouts/Profile/controls/NetworkDelegate.qml

```diff
@@ -0,0 +1,21 @@
+import StatusQ.Components 0.1
+import StatusQ.Core 0.1
+
+
+StatusListItem {
+    property var network
+    title: network.chainName
+    image.source: "..."
```

Comment:
Same comments here remove image.source, leftPadding and rightPadding

Reply:
done!, I keped the padding as by default they are not 0